### ISSUE TITLE
Timestamps on images are not readable

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#20402]
### Steps to reproduce

1. Enable message bubble layout
2. Send or receive any busy, grey-ish image
3. Hover over the image

### Outcome

![Screenshot 2022-01-04 at 15-05-57 Element general](https://user-images.githubusercontent.com/48614497/148117335-547329bb-85ed-4ecb-af6c-d181baf77a6b.png)

#### What did you expect?

To be able to read the timestamp.

#### What happened instead?

The timestamp is unreadable against the background of the image. Making the timestamp lighter and putting it against a dark backdrop, as in https://github.com/matrix-org/matrix-react-sdk/pull/6511, could aid readability.

### Operating system

NixOS unstable

### Browser information

Firefox 95.0.2

### URL for webapp

develop.element.io

### Application version

Element version: 211f48002759-react-00feb248778a-js-f30be8787934 Olm version: 3.2.8

### Homeserver

townsendandsmith.ml

### Will you send logs?

No

**Comment by amshakal** [comment]
I have added a gradient to the bottom of the image to take care of this issue. It should work for light and dark modes both.
<img width="458" alt="Screenshot 2022-01-12 at 4 28 54 pm" src="https://user-images.githubusercontent.com/85162587/149128744-762c29e8-8592-423f-b12d-f89be613fee8.png">

Link to Figma: https://www.figma.com/file/zVtAKK0hzKu5JkpbzlE1pG/Message-Bubbles?node-id=1691%3A38484
